e2e/wdio/headless/test.e2e.ts

```diff
@@ -100,15 +100,26 @@ describe('main suite 1', () => {
     it('should be able to handle successive scrollIntoView', async () => {
         await browser.url('http://guinea-pig.webdriver.io')
         await browser.setWindowSize(500, 500)
-        const searchInput = await $('.searchinput')
+        let searchInput = await $('.searchinput')
 
+        await browser.newWindow('http://guinea-pig.webdriver.io')
+        await browser.setWindowSize(500, 500)
+        const handles = await browser.getWindowHandles()
         const scrollAndCheck = async (params?: ScrollIntoViewOptions | boolean) => {
+
+            await browser.switchToWindow(handles[0])
+            await browser.pause(500)
+            searchInput = await $('.searchinput')
             await searchInput.scrollIntoView(params)
             await browser.pause(500)
             const [wdioX, wdioY] = await browser.execute(() => [
                 window.scrollX, window.scrollY
             ])
 
+            await browser.switchToWindow(handles[1])
```

Comment:
Instead of working with different window handles, can we just reload the page to reset the scroll position?